evals/eval.py

```diff
@@ -162,7 +163,36 @@ def worker_thread(args):
     def get_samples(self):
         if self.samples_jsonl is None:
             raise ValueError(
-                    "To use `get_samples`, you must provide a `samples_jsonl` path."
-                    "Got `None`.")
+                "To use `get_samples`, you must provide a `samples_jsonl` path." "Got `None`."
+            )
+
+        if exists_data_in_registry(self.samples_jsonl):
+            with open_by_file_pattern(self.samples_jsonl, "r") as f:
+                first_line = f.readline()
+            if "version https://git-lfs.github.com/spec/v1" in first_line:
+                logger.info(f"{self.samples_jsonl} is in git-lfs. Downloading...")
```

Comment:
Should we add an option (maybe env var) to toggle manual confirmation before download?